app/views/admin/developers/index.html.erb

```diff
@@ -0,0 +1,55 @@
+<div class="fixed-action-btn">
+  <%= link_to(new_admin_developer_path,
+              class: 'btn-floating btn-large tooltipped ',
+              'data-position' => 'left',
+              'data-tooltip' => 'Cadastrar um novo membro') do %>
+     <%= fa_icon "plus" %>
+  <% end %>
+</div>	
+
+<h1>Membros</h1>
+
+<table class="highlight responsive-table">
+  <thead>
+    <tr>
+      <th>Nome</th>
+      <th>Função</th>
+      <th>Ativo</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @developers.each do |dev| %>
+      <tr>
+        <td>
+          <%= link_to(admin_developer_path(dev)) do %>
+          <div class="developer-decoration">  
+            <%= image_tag attachment_url(dev, :profile_image, :fill, 50, 50, format: "jpg", fallback: "default-avatar"), :class => 'circle responsive-img index-image' %>   
+            <p><%= dev.name %></p>
+          </div>
+          <% end %>
+        </td>
+        <td><p><%= dev.function %></p></td>
+        <td>
+          <% if dev.active == true %>
```

Comment:
This can be done with a helper method.
```
  def humanize_boolean(boolean, {_true: nil, _false: })
    I18n.t((!!boolean).to_s)
  end
```
So in the view just use

```
<td><p><%= humanize_boolean(deve.active?) %></p></td>
```

config/locales/pt-BR.yml
```
'pt-BR':
  :true: 'Sim'
  :false: 'Não'
```